config/testgrids/kubernetes/kops/config.yaml

```diff
@@ -47,8 +47,6 @@ dashboards:
 - name: kops-gce
 - name: kops-grid
 - name: kops-distro-amzn2
-- name: kops-distro-centos7
-- name: kops-distro-centos8
 - name: kops-distro-deb9
```

Comment:
Probably this should go too:
```suggestion
```

Reply:
deb9 is still used in our grid jobs for kops <= 1.22, thats why the unit test didnt complain about it and why i kept it in `distro_images`

Reply:
ah, right